static/app/components/events/interfaces/nativeFrame.tsx

```diff
@@ -139,6 +145,20 @@ function NativeFrame({
     frame.function !== frame.rawFunction;
 
   const [expanded, setExpanded] = useState(expandable ? isExpanded ?? false : false);
+  const [isHovering, setHovering] = useState(false);
+
+  const contextLine = frame.context.find(l => l[0] === frame.lineNo);
+  const hasStacktraceLink = frame.inApp && !!frame.filename && (isHovering || expanded);
+  const hasStacktraceLinkInFrameFeatureFlag =
+    organization?.features?.includes('issue-details-stacktrace-link-in-frame') ?? false;
+  const showStacktraceLinkInFrame =
+    hasStacktraceLink && contextLine && hasStacktraceLinkInFrameFeatureFlag;
```

Comment:
We don't actually want to check for `contextLine` anymore. We actually want to check if the frame has a filename and lineno. That way we can display links for frames without source context.

Reply:
Yeah we should do it for both types